### DISCUSSION TITLE
Settings for curtains (cover) as easy as for media players

### DISCUSSION THREAD

**Discussion by 53l3cu5** [Clooos/Bubble-Card#652] [Feature requests]
the bubble card for audio media allows you to easily configure the buttons you want to appear or not. If it were possible to have the same functionality for the STOP button on the curtains, that would be nice. 

Also, as with the volume button for media, have a button that allows you to adjust the opening percentage of the curtain. 

And finally if it were possible to have one compact line rather than two lines.

It's just a suggestion because all this can be done using the slider type and using sub-buttons. But since you already did the work for the Media Player, it would be nice to apply the reasoning to the curtains

**Comment by Clooos** [comment]
Hi, I'm planning to rework the cover card since the v2 release, your suggestions are great 👌
